plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/variables/SQLVariablesPanel.java

```diff
@@ -362,11 +364,10 @@ public void run() {
                 @Override
                 public void run() {
                     if (!varsTable.getSelection().isEmpty()) {
-                        Object varElement = ((IStructuredSelection) varsTable.getSelection()).getFirstElement();
-                        if (varElement instanceof DBCScriptContext.VariableInfo) {
-                            RemoveVariableAction action = new RemoveVariableAction(mainEditor, ((DBCScriptContext.VariableInfo) varElement).name);
-                            action.run();
-                        }
+                        final StructuredSelection selection = (StructuredSelection) varsTable.getSelection();
+                        List<String> varsList = Arrays.stream(selection.toArray()).map(el -> ((DBCScriptContext.VariableInfo) el).name).collect(Collectors.toList());
```

Comment:
Unnecessary usage of Stream API.

The `selection` variable could be inlined into a `for-loop` that will accumulate variable names into a list.